src/command.rs

```diff
@@ -227,83 +276,34 @@ impl Context {
 
         let mut locked = Vec::new();
 
-        #[async_recursion]
-        async fn traverse_and_install(
```

Comment:
I would leave it in here to make it clear that it is not usable in the outside world

Reply:
As it is not `pub`, it is neither visible in docs nor usable outside of this module. As it operates on the current `buffrs` package, I made it a method. Do you prefer the inlined version?

Reply:
Yes – they communicate two things, the inlined version communicates that this is only ever relevant to the `install` fn – the private version indicates that this is usable for other commands which it isnt